catroid/src/main/java/org/catrobat/catroid/ui/fragment/AddUserDataToUserDefinedBrickFragment.java

```diff
@@ -95,8 +95,32 @@ public View onCreateView(@NotNull LayoutInflater inflater, ViewGroup container,
 		TextView addUserDataUserBrickTextView = view.findViewById(R.id.brick_user_defined_add_user_data_description);
 
 		addUserDataUserBrickEditText.setText(userBrickTextView.getText());
+		InputWatcher.TextWatcher textWatcher = new InputWatcher.TextWatcher() {
+			@Override
+			protected boolean isNameUnique(String name) {
```

Comment:
I think you don't need to override this function, as the same functionality is already provided by the base class.